mscore/musescore.cpp

```diff
@@ -2990,6 +2993,19 @@ void MuseScore::clipboardChanged()
       }
 
 //---------------------------------------------------------
+//   inputMethodLocaleChanged
+//---------------------------------------------------------
+
+void MuseScore::inputMethodLocaleChanged()
+      {
+      QInputMethod* im = QGuiApplication::inputMethod();
+      _qInputMethodScript = im->locale().script();
+      qDebug("new input method locale: %d", _qInputMethodScript);
+      if (_qInputMethodScript == QLocale::JapaneseScript)
+            qDebug("Might need to handle japanese IME special chars");
+      }
```

Comment:
note, I just tested having the windows IME initially be set to japanse before starting musescore, I notice that upon closing start center, the IME gets set to LatinScript.  I also notice that if I alt tab away from musescore, and change the input language with WinKey+Space, that this signal received upon AltTabbing back into musescore.